lib/resources/r4/document_reference.yaml

```diff
@@ -440,9 +440,9 @@ fields:
   type: Code
   action:
     - update
-  description: Identifies the type of the data in the attachment and allows a method to be chosen to interpret or render the data. Includes mime type parameters such as charset where appropriate.
+  description: Identifies the type of the data in the attachment and allows a method to be chosen to interpret or render the data. Includes MIME type parameters such as charset where appropriate.
```

Comment:
thoughts on code for `charset`?

Reply:
Not opposed, but we don't currently code that throughout all the .yaml files in the repository.
it's only backtick'd when it's part of a longer phrase.
e.g. 
>   note: Must be provided. Content Type must include a supported MIME type and character set. The supported values are `application/pdf`, `text/plain;charset=utf-8`, `text/richtext;charset=utf-8`, `text/rtf;charset=utf-8`, `text/html;charset=utf-8`, `application/xml;charset=utf-8`, and `application/xhtml+xml;charset=utf-8`

Sourced in this same document:
https://github.com/cerner/fhir.cerner.com/blob/b1d03f7c0a09863de603dcff2f68ed0d612af975/lib/resources/r4/document_reference.yaml#L200